build.sbt

```diff
@@ -16,16 +19,16 @@ assemblyOption in assembly ~= {
   _.copy(cacheUnzip = true)
 }
 
-val pureconfigVersion = "0.8.0"
+val pureconfigVersion = "0.12.1" //didn't work with 0.11.1 but works with this strangely
```

Comment:
This comment doesn't add value outside this review. Please remove.

Reply:
Okay. Would you suggest rephrasing this type of comments so that they have more insight or leave out comments like these entirely?

Reply:
The comment only has value for the PR - so it shouldn't be in the code. Let's remove this.

Reply:
Sure thing. Thank you!